nginx/config.go

```diff
@@ -188,7 +188,8 @@ http {
 		vhost_traffic_status_filter_by_set_key {{ $appConfig.Name }} application::*;
 
 		location / {
-			{{ if $appConfig.Available }}proxy_buffering off;
+			{{ if $appConfig.Maintenance }}return 503 {{ $appConfig.MaintenancePage }};{{ else if $appConfig.Available }}
+			proxy_buffering off;
```

Comment:
Nit... if we care about how the generated configuration looks, we might want to move `proxy_buffering off` up to the end of the previous line. That will avoid an empty line after `location / {` in the event that `$appConfig.Available` is `true`.